src/core/ProgressBar/__tests__/ProgressBar.test.tsx

```diff
@@ -5,9 +5,29 @@ import { ProgressBar } from '../ProgressBar';
 
 describe('ProgressBar', () => {
   it('Renders', () => {
+    function checkProgressValue(value: number) {
```

Comment:
I don't think is check is correct...this is checking values passed in the test file...the test file should check the component logic.

Reply:
@arkajyotiMukherjee I need to check to make sure if the test will work as expected.

Reply:
This is checking if the value passes in the test case is correct or not...before it is passed...
It should rather pass values of all sort of edge cases and check if the component is behaving as expected
@Hasmik8